financeextras.php

```diff
@@ -305,3 +305,35 @@ function financeextras_civicrm_selectWhereClause($entity, &$clauses) {
     $hook->filterBasedOnOwnerOrganisations($ownerOrganisationToFilterIds);
   }
 }
+
+/**
+ * Implements hook_civicrm_links().
+ */
+function financeextras_civicrm_links($op, $objectName, $objectId, &$links, &$mask, &$values) {
```

Comment:
both of these two hooks: `financeextras_civicrm_links` and `financeextras_civicrm_buildForm` exist already on the workstream branch, so you need to incorporate the stuff here with the existing implementation.